.gitlab-ci.yml

```diff
@@ -229,6 +229,23 @@ build:bleeding-edge:
     EXTRA_CONF: "-native-compiler yes -coqide opt"
     EXTRA_PACKAGES: "$COQIDE_PACKAGES"
 
+build:windows:
+  stage: build
```

Comment:
Might be better in stage: test to avoid holding those jobs up.

Reply:
The artifact upload seems to take a while so definitely put the job in the most parallel stage.

Reply:
I also thought I could do a later `package` stage. Especially because it is going to be required on macOS anyway, I think. Does it sound good?

Reply:
>I also thought I could do a later package stage. Especially because it is going to be required on macOS anyway, I think. Does it sound good?

It's less parallel, what's the benefit? Also if any of the tests break it will block the package stage.
Why would it be required on apple?